test/addons/rbd-mirror/start-data/vrc-sample.yaml

```diff
@@ -6,6 +6,9 @@ apiVersion: replication.storage.openshift.io/v1alpha1
 kind: VolumeReplicationClass
 metadata:
   name: vrc-sample
+  labels:
+    ramendr.openshift.io/storageid: rook-ceph-$cluster-1
+    ramendr.openshift.io/replicationid: rook-ceph-$cluster-1
```

Comment:
RepID should be common across the clusters, so naming cannot just use local cluster name. Maybe use, `cluster1-cluster2` (in the same order across the clusters) to craft the repID.

Reply:
or maybe use `ramendr.openshift.io/replicationid: rook-ceph-provider-1`, preparing for having more than one.

Reply:
i have used `replication-1` as the value.